controllers/nstemplateset/nstemplateset_controller.go

```diff
@@ -95,6 +95,7 @@ type Reconciler struct {
 //+kubebuilder:rbac:groups=quota.openshift.io,resources=clusterresourcequotas,verbs=get;list;watch;create;update;patch;delete
 //+kubebuilder:rbac:groups=networking.k8s.io,resources=networkpolicies,verbs=get;list;watch;create;update;patch;delete
 //+kubebuilder:rbac:groups=dbaas.redhat.com,resources=dbaastenants,verbs=get;list;watch;create;update;patch;delete
```

Comment:
we could drop this
```suggestion
```

Reply:
ok, let me do that. I did not realize it was not needed anymore